keras_nlp/layers/masked_lm_mask_generator.py

```diff
@@ -42,22 +42,24 @@ class MaskedLMMaskGenerator(keras.layers.Layer):
         mask_selection_rate: float, the probability of a token is selected for
             masking.
         mask_token_id: int. The id of mask token.
-        mask_selection_length: int, defaults to None. Maximum number of tokens
+        mask_selection_length: int. Maximum number of tokens
             selected for  masking in each sequence. If set, the output
             `mask_positions`, `mask_ids` and `mask_weights` will be padded
             to dense tensors of length `mask_selection_length`,
             otherwise the output will be a RaggedTensor.
-        unselectable_token_ids: A list of tokens, defaults to [0] (the default
-            `padding_token_id`).
-        mask_token_rate: float, defaults to 0.8. `mask_token_rate` must be
+            Defaults to `None`.
+        unselectable_token_ids: A list of tokens, [0] is default
```

Comment:
remove this first reference to default

Reply:
Not sure what language you want used here

Reply:
I guess I would do this ->

unselectable_token_ids: A list of tokens id that should not be considered eligible for masking. Defaults to `[0]`, where 0 is assumed to be the padding token id.

Would that run counter to what you are trying to do?

Reply:
Yeah because its ambiguous whether the default value becomes
> `[0]`, where 0 is assumed to be the padding token id

Or
> `[0]`

PS: Maybe some kind of `itemgetter(0)` would make more sense as the default here? - Just trying to think of this in SQL, CLI, and SDK contexts.

Reply:
I'm not sure how `itemgetter(0)` would work. This isn't an argument we use to index into something, this is a set of values we use to compare against similarly tokenized input in this same integer space.
 
Maybe just say

```
unselectable_token_ids: A list of tokens id that should not be considered eligible for masking. By default we assume `0` corresponds to a padding token and ignore it. Defaults to `[0]`.
```

A little redundant but keeps the consistent form on the last sentence.

Reply:
Done